scripts/audit_branches.py

```
@@ -0,0 +1,98 @@
+                if file not in branch_files:
+                    issues.append(f"Missing {file}")
+        else:
+            issues.append("Could not list branch files")
+
+        # 2. Check for security helpers in src/utils/navigation.ts
+        # We read the file content directly from the git object database.
+        nav_content = run_command(["git", "show", f"{branch}:src/utils/navigation.ts"])
+        if nav_content:
+            for helper in SECURITY_HELPERS:
+                if helper not in nav_content:
+                    issues.append(f"Missing security helper: {helper}")
```

Ruff (0.15.7)

[warning] Use `list.extend` to create a transformed list

(PERF401)